Documentation/SwiftlyDocs.docc/use-tools-different-sources.md

```diff
@@ -0,0 +1,39 @@
+# Access Tools from Different Sources
+
+Access tools that can be from different sources.
+
+swiftly installs and manages toolchains for you that are available from swift.org. It sets up your shell environment to work well with these. There are other sources of Swift toolchains, and the common tools found in them, too. How do I access these from within my swiftly environment?
+
+Also, sometimes it's difficult to determine whether there is a bug in swiftly itself, or the bug lies in the particular toolchain at swift.org. How can I figure this out?
```

Comment:
This articles seems to be trying to serve two masters - how to use the new `link` and `unlink` subcommands as well as how to wrangle seeing and controlling where a specific tool is being invoked from. I think you may be trying to go too broad here. This reads to me like two focused articles sort of mashed into one.